setup.py

```diff
@@ -15,7 +15,7 @@
 from setuptools import find_packages, setup
 
 
-VERSION = "0.11.2.dev0"
+VERSION = "0.12.0"
```

Comment:
This can be automated actually: https://github.com/huggingface/diffusers/blob/2c25b98c8ea74cfb5ec56ba49cc6edafef0b26af/Makefile#L86

Reply:
Yes, we can take a look into this at some point.